.github/workflows/publish-new-version.yaml

```
@@ -14,10 +14,10 @@ on:
       version:
         description: What version number would you like to use? The version number should be **without** a leading `v`, e.g. `5.7.1` or `6.2.4`.
       bump-main:
-        description: Should we bump the package.json main version?
+        description: Only check it if you're releasing from a branch and you want to backport the version to main
```

Maybe change the wording a bit here?
```suggestion
        description: I'm releasing from a branch and want the version number backported to main
```